src/utils.py

```diff
@@ -0,0 +1,55 @@
+from nes_py.wrappers import JoypadSpace
+import gym_super_mario_bros
+from gym_super_mario_bros.actions import SIMPLE_MOVEMENT
+import numpy as np
+import matplotlib.pyplot as plt
+from IPython import display as ipythondisplay
+from tqdm import tqdm
+from pyvirtualdisplay import Display
+
+class Utils:
+
+    def __init__(self, level):
+        # add level selection
+        self.env = gym_super_mario_bros.make('SuperMarioBros-1-1-v0')
```

Comment:
The environment string 'SuperMarioBros-1-1-v0' can be a variable that's passed in (in case you all want to change this later)